draft-barnes-hpke-hpke.md

```diff
@@ -391,7 +404,7 @@ computation of the public key using the private key, or just syntax
 expressing the retrieval of the public key, assuming it is stored along
 with the private key object.
 
-The following two functions are defined to facilitate domain separation of
+The following functions are defined to facilitate domain separation of
 KDF calls as well as context binding:
```

Comment:
I would split this up with separate code blocks for two-stage and one-stage KDFs.